site/profile/manifests/core/icinga_agent.pp

```diff
@@ -63,10 +64,11 @@
   }
   ##Icinga2 feature API config
   class { '::icinga2::feature::api':
-    pki             => 'puppet',
```

Comment:
The default value for pki is "icinga2" but your setting it to `none` in the icinga_master profile, are they intentionally different values?

 https://github.com/Icinga/puppet-icinga2/blob/master/manifests/feature/api.pp#L149

Reply:
this was the original recommendation from the repo, but couldn't make it work earlier. The master identify itself as CA.